HandleNHNotificationCallOrchestrator/__tests__/handler.test.ts

```diff
@@ -0,0 +1,88 @@
+/* tslint:disable:no-any */
+// tslint:disable-next-line: no-object-mutation
+process.env = {
+  ...process.env,
+  AZURE_NH_ENDPOINT:
+    "Endpoint=sb://io-lab-notification-hub-ns.servicebus.windows.net/;SharedAccessKeyName=DefaultFullSharedAccessSignature;SharedAccessKey=C4xIzNZv4VrUnu5jkmPH635MApRUj8wABky8VfdPLqg=",
```

Comment:
hope it's not a working connection string :)

Reply:
Nope it's not working right @BurnedMarshal ?

Reply:
I don't know, that is the key that provides to me @gunzip for testing purpose

Reply:
it's a real service, plase remove it :)